### ISSUE TITLE
Releases panel is removed (without `releases-tab`)

### ISSUE THREAD

**Issue by altrisi** [sindresorhus/refined-github#3848]
Edit: This (not being able to access releases at all with only one of the features enabled) has been confirmed as WAI. There is a workaround userscript here: https://github.com/sindresorhus/refined-github/issues/3848#issuecomment-751515243.

---

The releases panel on the right of every repo is removed (since one of the latter updates of the extension), even when `releases-tab` is disabled.

The panel is present when disabling the extension.

This happens in any repo (with releases, at least).

Example URL where the bug appears: https://github.com/sindresorhus/refined-github

Screenshot:
Extension disabled:
![imagen](https://user-images.githubusercontent.com/17107132/103175313-559c3a80-4869-11eb-9ad8-75f5ee1c0b67.png) 
Extension enabled:
![imagen](https://user-images.githubusercontent.com/17107132/103175319-65b41a00-4869-11eb-93c7-64252b147642.png)

**Comment by yakov116** [comment]
A few features were merged. To unhide that disable `clean-repo-sidebar` https://github.com/sindresorhus/refined-github/pull/3783

**Comment by altrisi** [comment]
But, shouldn't that feature hide only empty panels?

**Comment by yakov116** [comment]
Well we are showing the release tab as part of the other tabs.

**Comment by altrisi** [comment]
Except when `releases-tab` is disabled, like in this issue, or is it present somewhere else?

**Comment by busches** [comment]
@altrisi we don't have a rule to not hide that section when the releases tab feature is disabled. It's possible to add it, but this type of interaction occurs with plenty of features and we don't code around them either. I'm not in favor of coupling them together and instead letting the user disable both.

**Comment by fregante** [comment]
That part of `releases-tab` was consolidated into `clean-repo-sidebar` as requested in https://github.com/sindresorhus/refined-github/issues/3706

**Comment by altrisi** [comment]
Then, there is no way except from manually changing the url of getting into the releases page without enabling `releases-tab` or disabling `clean-repo-sidebar`.

**Comment by fregante** [comment]
Correct. Either leave both enabled or disable both.

**Comment by altrisi** [comment]
That makes no sense to me, but well, it's your extension. I suppose it's ok to make things not work at all if that's what you want.

I can't find where it was requested, though.

**Comment by fregante** [comment]
Yesterday `clean-repo-sidebar` didn’t even exist, now you’re complaining that you want to keep it enabled. 

And why do you want to disable `releases-tab`?

**Comment by fregante** [comment]
It was discussed in https://github.com/sindresorhus/refined-github/pull/3566#issuecomment-700611153

**Comment by altrisi** [comment]
> Yesterday `clean-repo-sidebar` didn’t even exist, now you’re complaining that you want to keep it enabled.

Yesterday `clean-repo-sidebar` didn't even exist, true, but `hide-empty-meta` did, which had features that were moved into `clean-repo-sidebar` (and some from `hide-zero-packages` too).

> And why do you want to disable `releases-tab`?

I just don't like it, I think it gets too much importance there. I think that's the point of being able to choose features.

Anyway, don't worry, I'll live with it, or maybe at some point make myself a script to allow both somehow.

**Comment by fregante** [comment]
> I think that's the point of being able to choose features.

Right, that’s why they been split. If we merge them someone will ask for the opposite. 

The defaults work for all, so you can’t say that they don’t make sense. If you disable features that’s your own choice.

**Comment by FranklinYu** [comment]
> And why do you want to disable releases-tab?

I wouldn’t mind having an additional Releases tab, but “viewing the latest release” is a very frequent action for me, so I want to be able to do that without an additional navigation. That said, I understand the complication of interaction between rules, so I’d simply live with it (disabling `clean-repo-sidebar` is not too bad).

**Comment by fregante** [comment]
As you may have noticed, the Releases section is back in the sidebar since version 21.2.1
